src/engine/strat_engine/thinpool/filesystem.rs

```diff
@@ -28,14 +27,15 @@ use super::super::cmd::{create_fs, set_uuid, xfs_growfs};
 use super::super::dm::get_dm;
 use super::super::names::{format_thin_ids, ThinRole};
 use super::super::serde_structs::FilesystemSave;
+use super::thinpool::{DATA_BLOCK_SIZE, DATA_LOWATER};
 
 const DEFAULT_THIN_DEV_SIZE: Sectors = Sectors(2 * IEC::Gi); // 1 TiB
 
 const TEMP_MNT_POINT_PREFIX: &str = "stratis_mp_";
 
 /// TODO: confirm that 256 MiB leaves enough time for stratisd to respond and extend before
 /// the filesystem is out of space.
-pub const FILESYSTEM_LOWATER: Sectors = Sectors(256 * IEC::Mi / (SECTOR_SIZE as u64)); // = 256 MiB
+pub const FILESYSTEM_LOWATER: Sectors = Sectors(DATA_LOWATER.0 * DATA_BLOCK_SIZE.0 * 4);
```

Comment:
I'd also update the comment, TODO no longer valid.

Reply:
thanks